governance.md

```diff
@@ -24,17 +28,53 @@ Every code owner is expected to:
 * Approve only those PRs in which the contributor made the requested improvements
 * Check if the related CI tests have completed successfully before approving the PR
 * Make sure that the PR approval flow runs smoothly
-* Prioritize issues and manage their processing
 * Proactively fix bugs
 * Perform maintenance tasks for their projects
-
-### Add or remove a code owner
+* Issue that is assigned to current release is treated with priority by maintainer that is an assignee
+* Monitor [Slack](http://slack.kyma-project.io)
+* Triage GitHub issues and perform pull request reviews for other maintainers and the community. Follow [labels workflow](/governance/#issues-workflow-issues-workflow) document
+* In general continue to be willing to spend at least 50% of one's time working on Kyma (~2.5 business days per week).
+
+### Process for becoming a maintainer aka codeowner
```

Comment:
correct me if I'm wrong but to be a Kyma project member we also require the 2FA? if yes, then we should as sth like:

- Enabled [two-factor authentication](https://help.github.com/en/articles/about-two-factor-authentication) on GitHub account